Makefile

```diff
@@ -34,39 +30,26 @@ style: flake8 isort license
 
 ## Run the tests.
 test:
-	-docker-compose run --rm -e ENVIRONMENT=testing -e POSTGRES_DB_NAME=testing web \
+	-docker-compose run --rm -e ENVIRONMENT=testing web \
 		pytest --cov=src/warehouse tests
 
 ## Run the tests and report coverage (see https://docs.codecov.io/docs/testing-with-docker).
 test-travis:
 	$(eval ci_env=$(shell bash <(curl -s https://codecov.io/env)))
-	docker-compose run --rm -e ENVIRONMENT=testing $(ci_env)  web \
+	docker-compose run --rm -e ENVIRONMENT=testing $(ci_env) web \
```

Comment:
Wasn't the latest pattern to mount a volume for the coverage file and then run the codecov bash script afterwards?

Reply:
Ok - I will make a PR in the cookiecutter.